tests/integration/test_scaling_monolithic.py

```diff
@@ -23,13 +25,13 @@
 @pytest.mark.abort_on_fail
 async def test_deploy_tempo(ops_test: OpsTest):
     tempo_charm = await ops_test.build_charm(".")
-    resources = {"tempo-image": METADATA["resources"]["tempo-image"]["upstream-source"]}
-    await ops_test.model.deploy(tempo_charm, resources=resources, application_name=APP_NAME),
+    await ops_test.model.deploy(tempo_charm, application_name=APP_NAME)
 
     await ops_test.model.wait_for_idle(
         apps=[APP_NAME],
-        # tempo might be in waiting as it waits for tempo workload ready
-        raise_on_blocked=True,
+        # coordinator will be blocked on s3 and workers integration
+        status="blocked",
+        raise_on_blocked=False,
```

Comment:
I _think_ if you expect status="blocked" it should also implicitly not raise on blocked.